cloudbuild.yaml

```diff
@@ -3,17 +3,22 @@
 timeout: 1200s
 options:
   substitution_option: ALLOW_LOOSE
+  machineType: N1_HIGHCPU_8
 steps:
-  - name: gcr.io/cloud-builders/docker
+  - name: gcr.io/k8s-testimages/gcb-docker-gcloud:v20201130-750d12f
+    entrypoint: bash
+    env:
+      - TAG=$_GIT_TAG
+      - HOME=/root  # for docker buildx
     args:
-      - build
-      - --tag=gcr.io/$PROJECT_ID/security-profiles-operator:$_GIT_TAG
-      - --tag=gcr.io/$PROJECT_ID/security-profiles-operator:latest
-      - .
+    - -c
+    - |
+      gcloud auth configure-docker && \
+      make image-cross
 substitutions:
   # _GIT_TAG will be filled with a git-based tag for the image, of the form
   # vYYYYMMDD-hash, and can be used as a substitution
   _GIT_TAG: '12345'
 images:
-  - 'gcr.io/$PROJECT_ID/security-profiles-operator:$_GIT_TAG'
-  - 'gcr.io/$PROJECT_ID/security-profiles-operator:latest'
+  - 'gcr.io/$PROJECT_ID/security-profiles-operator-amd64:$_GIT_TAG'
+  - 'gcr.io/$PROJECT_ID/security-profiles-operator-amd64:latest'
```

Comment:
@saschagrunert I didn't quite follow this change, could you provide some more context?

Reply:
This is just a check after the cloudbuild.yaml has run. It verifies that those images exist in the registry, nothing more.

Reply:
Gotcha, thanks! That makes sense, wasn't aware it didn't work with manifests :)

Reply:
> It verifies that those images exist in the registry, nothing more.

Doesn't this actually _push_ those images from the docker daemon to the registry? https://cloud.google.com/build/docs/build-config#images

In this case, if you've already pushed them, I guess the push is a no-op?

Reply:
That’s not doing the actual push. As I tested it, it was just checking that the images are now available (aka already pushed).